skrf/frequency.py

```diff
@@ -208,7 +208,6 @@ def __getitem__(self, key: Union[str, int, slice]) -> 'Frequency':
         if isinstance(key, str):
 
             # they passed a string try and do some interpretation
-            re_numbers = re.compile(r'.*\d')
```

Comment:
Why does this line have been removed? Wasn't used after?

Reply:
No, it was not used in the file